cogs/respects.py

```diff
@@ -202,8 +202,9 @@ def checkIfUserPaidRespect(self, ctx):
             if self.settings[sid][cid][KEY_MSG]:
                 try:
                     await self.bot.delete_message(self.settings[sid][cid][KEY_MSG])
-                except: # pylint: disable=bare-except
-                    pass
+                except:

```

Comment:
We can replace this bare except to catch these two:
```suggestion
                except (discord.Forbidden, discord.NotFound):
```